### PR TITLE
rename: invalid commands removed

### PR DESCRIPTION
**Pull request by aoristos**
Hello,
Thank you for the project. I'm fairly new at github: It's the first time I've started a pull request.
I suggest the previous commands to be removed because they are no valid rename commands (they look more like vim commands).
I've added some valid rename command (that at least worked for me)
Any suggestion welcome.
grz,

### CODE REVIEW

File: dnf+ [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
```
Comment (EmilySeville7cfg) [inline dnf+:1]: - Optional change: wrong filename: `dnf` is expected.
- Common page flaws are not listed for other pages.

File: ffmpeg+ [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# To print file metadata:
```
Comment (EmilySeville7cfg) [inline ffmpeg+:1]: Optional change: wrong filename: `ffmpeg` is expected.

File: find+ [hunk 3]
```diff
@@ -0,0 +1,106 @@
+#WIP
```
Comment (EmilySeville7cfg) [inline find+:1]: Optional change: wrong filename: `find` is expected.

File: grep+ [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# To search a file for a pattern:
```
Comment (EmilySeville7cfg) [inline grep+:1]: Optional change: wrong filename: `grep` is expected.

File: ranger+ [hunk 5]
```diff
@@ -0,0 +1,67 @@
+# Ranger Cheatsheet
```
Comment (EmilySeville7cfg) [inline ranger+:1]: Optional change: wrong filename: `ranger` is expected.

File: rename+ [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# To replace 'sometext' with 'replacedby' in all file names in current directory:
```
Comment (EmilySeville7cfg) [inline rename+:1]: Optional change: wrong filename: `rename` is expected.

File: rsync+ [hunk 7]
```diff
@@ -0,0 +1,44 @@
+# To copy (update) the directory name and all the content of /src/foo into destination /dest/foo
```
Comment (EmilySeville7cfg) [inline rsync+:1]: Optional change: wrong filename: `rsync` is expected.

File: vim+ [hunk 8]
```diff
@@ -0,0 +1,251 @@
+# ---
```
Comment (EmilySeville7cfg) [inline vim+:1]: Optional change: wrong filename: `vim` is expected.

File: dnf+ [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+tags: [ packaging ]
+---
+# To install a package:
+dnf install <package>
```
Comment (EmilySeville7cfg) [inline dnf+:5]: Multiple fixes available: please chose one of them.

```suggestion
# Install a specific package:
dnf install "<package>"
```

- Optional change: add sample input and output right after this example. It's not a TlDr page where u have a more restricted and compact page format.
- Optional change: it's safer to use quotes.
- Optional change: `to` can be safely removed.
- Optional change: `a specific` may sound better.

```suggestion
# Install specific packages:
dnf install "<package1 package2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple packages at once for `install` command then u can use this fix.

File: dnf+ [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+# To find which package provides an executable:
+dnf provides <executable>
+
+# The following are available after installing "dnf-plugins-core"
+
+# To download a package:
+dnf download <package>
+
+# To install the build dependencies for a SRPM or from a .spec file:
```
Comment (EmilySeville7cfg) [inline dnf+:18]:
```suggestion
# Install the build dependencies for a SRPM or from a specific .spec file:
```

Almost the same as above.

File: dnf+ [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+dnf builddep <file>
```
Comment (EmilySeville7cfg) [inline dnf+:19]:
```suggestion
dnf builddep <path/to/file>
```

Optional change: it's better to use `<path/to/file>` format.

File: dnf+ [hunk 12]
```diff
@@ -0,0 +1,19 @@
+---
+tags: [ packaging ]
+---
+# To install a package:
+dnf install <package>
+
+# To find packages matching <phrase>:
+dnf search <phrase>
```
Comment (EmilySeville7cfg) [inline dnf+:8]: Multiple fixes available: please chose one of them.

```suggestion
# Find packages matching a specific phrase:
dnf search "<phrase>"
```

Almost the same as above.

```suggestion
# Find packages matching one of specific phrases:
dnf search "<phrase1 phrase2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple phrases at once for `search` command then u can use this fix.

File: dnf+ [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+
+# To find which package provides an executable:
+dnf provides <executable>
```
Comment (EmilySeville7cfg) [inline dnf+:11]: Multiple fixes available: please chose one of them.

```suggestion
# Find which package provides a specific executable:
dnf provides "<executable>"
```

Almost the same as above.

```suggestion
# Find which package provides one of specific executables:
dnf provides "<executable1 executable2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple executables at once for `provides` command then u can use this fix.

File: dnf+ [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+
+# The following are available after installing "dnf-plugins-core"
+
+# To download a package:
+dnf download <package>
```
Comment (EmilySeville7cfg) [inline dnf+:16]: Multiple fixes available: please chose one of them.

```suggestion
# Download a specific package:
dnf download "<package>"
```

Almost the same as above.

```suggestion
# Download specific packages:
dnf download "<package1 package2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple packages at once for `download` command then u can use this fix.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found. That's why PR was approved.
- This page should be reviewed by `dnf`, `ffmpeg`, `ranger`, `rename`, `rsync` users.
- Thanks for a contribution! ❤️

**Comment by chrisallenlane** [comment]
Hi, @aoristos 

Thanks for the effort you put into this.

I like some of the changes contained here, but would prefer not to merge others.

I'm going to close this PR without merging, but if you have the time and interest, I'll ask that you open another PR containing your `ranger` cheatsheet. That's great, and the project would benefit from its inclusion :+1:

**Pull request closed without merging**